bearer/bearer.go

```diff
@@ -352,17 +354,6 @@ func (b Token) SigningKeyBytes() []byte {
 
 // ResolveIssuer resolves issuer's user.ID from the key used for Token signing.
 // Returns zero user.ID if Token is unsigned or key has incorrect format.
-//
-// See also SigningKeyBytes.
 func (b Token) ResolveIssuer() user.ID {
-	var usr user.ID
-	binKey := b.SigningKeyBytes()
-
-	if len(binKey) != 0 {
-		if err := user.IDFromKey(&usr, binKey); err != nil {
-			usr = user.ID{}
-		}
-	}
-
-	return usr
+	return b.issuerID
```

Comment:
Because this can't be done. You can get this token from the network and it doesn't have any `issuer` inside, it's derived from the key. Which is a bug at the API level: nspcc-dev/neofs-api#266

We can't do much about it right now at the same time.

Reply:
Ow, ok, removed this commit.